connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSourceTaskTest.java

```diff
@@ -391,37 +369,26 @@ public void testFailureInPoll() throws Exception {
         taskFuture.get();
         assertPollMetrics(0);
 
-        PowerMock.verifyAll();
+        verifyCleanStartup();
+        verifyOffsetFlush(true);
+        verify(statusListener).onFailure(taskId, exception);
+        verify(sourceTask).stop();
```

Comment:
Looks like we might be missing coverage for this call (originally verified in `expectEmptyOffsetFlush`):
```suggestion
        verify(sourceTask).commit();
        verify(sourceTask).stop();
```

Reply:
Believe this still needs to be addressed?

Reply:
Ah, never mind. This is covered by changes to `verifyOffsetFlush`.